docs/hooks/useIoBrokerState.md

```diff
@@ -92,7 +92,7 @@ const MyComponent: React.FC<MyAppProps> = (props) => {
 import { useIoBrokerState } from "iobroker-react/hooks";
 
 const MyComponent: React.FC<MyAppProps> = (props) => {
-	const [, myStateAck, setMyState] = useIoBrokerState({
+	const [myState, myStateAck, setMyState] = useIoBrokerState({
```

Comment:
myState isn't used. No need to declare it here, or what am I missing?

Reply:
Yes, you are right, I overlooked that.